### PR TITLE
feat: Onboard credit score issuer by supplying them the DL, DL Assurance

### LINKED ISSUE #571: Onboard credit score issuer by supplying them the DL, DL Assurance (no login)

### PR DESCRIPTION
**Pull request by fqutishat**
closes #571

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/scripts/strapi_configure.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ fi
 # Add user data
 result=$(curl --header "Content-Type: application/json" --header "Authorization: Bearer $token" \
    --request POST \
-   --data '{"userid":"100","name":"Foo Bar","email":"foo@bar.com","type":"user"}' \
+   --data '{"userid":"100","name":"John Smith","email":"foo@bar.com","type":"user"}' \
```
Comment (rolsonquadras) [inline test/bdd/fixtures/scripts/strapi_configure.sh:98]: Should we update email as well ? Something like `john.smith@example.com` ?

File: cmd/issuer-rest/static/creditscore.html [hunk 2]
```diff
@@ -76,12 +80,7 @@ <h5 class="w-full my-2 text-xl leading-tight left text-black">Find the services
             <div class="flex-none mt-auto bg-white rounded-b rounded-t-none overflow-hidden shadow p-6">
                 <img class="object-contain h-64 w-full" src="img/credit_score.jpg">
                 <div class="flex justify-center mt-8 pt-2">
-                    <form id="didCommDemo"  action="">
-                        <input type="text" id="demoType" name="demoType" value="" checked hidden>
-                        <input type="text" id="didCommScope" name="didCommScope" value="" checked hidden>
-                        <input type="text" id="adapterProfile" name="adapterProfile" value="" checked hidden>
-                        <input type="submit" id="creditScore" class="lg:mx-0 hover:underline credit-gradient text-white font-bold my-6 py-4 px-8 shadow-lg" value="Get your credit report">
-                    </form>
+                    <input type="submit" id="getDrivingLicense" onclick="getDrivingLicense()" class="lg:mx-0 hover:underline credit-gradient text-white font-bold my-6 py-4 px-8 shadow-lg" value="Get your credit report">
```
Comment (rolsonquadras) [inline cmd/issuer-rest/static/creditscore.html:83]: Just wondering if we should keep the existing `credit-score` issuer as is (`New Bank Account` use case) and create a new `credit-score` issuer with DL creds (`Government RP` use case).

File: test/bdd/fixtures/scripts/strapi_configure.sh (hunk 1 above)
Comment (fqutishat) [reply test/bdd/fixtures/scripts/strapi_configure.sh]: done

File: cmd/issuer-rest/static/creditscore.html (hunk 2 above)
Comment (fqutishat) [reply cmd/issuer-rest/static/creditscore.html]: done add new page creditscorenologin.html

**Review by rolsonquadras — APPROVED** [review]

File: cmd/issuer-rest/static/creditscore.html [hunk 3]
```diff
@@ -134,4 +134,3 @@ <h5 class="w-full my-2 text-xl leading-tight left text-black">Find the services
 </body>
```
Comment (talwinder50) [inline cmd/issuer-rest/static/creditscore.html:135]: Why this is empty html file?

**Review by sudeshrshetty — APPROVED** [review]

**Review by talwinder50 — APPROVED** [review]

**Pull request merged**